src/core/get-invalid-colors-of-file.ts

```diff
@@ -8,6 +9,11 @@ export const getInvalidColorsOfFile = (
   palette: ColorPalette,
   fileName: string
 ): InvalidColorResult[] => {
+  if (palette.exclude) {
```

Comment:
Please combine this if conditions to a single condition:

```ts
if (palette.exclude?.some((pattern) => minimatch(fileName, pattern))) {
```

Reply:
Fixed